src/saving/SaveList.cs

```diff
@@ -137,6 +140,8 @@ public override void _Process(float delta)
                 if (SelectableItems)
                     item.Connect(nameof(SaveListItem.OnSelectedChanged), this, nameof(OnSubItemSelectedChanged));
 
+                item.Connect(nameof(SaveListItem.OnDoubleClicked), this, nameof(OnItemDoubleClicked));
```

Comment:
Instead put the value as a connection parameter here, similarly to how the `new Array { save }` is defined for the other connections below this one.

Reply:
Not fully understand that.

Reply:
```
                 item.Connect(nameof(SaveListItem.OnDoubleClicked), this, nameof(OnItemDoubleClicked), new Array { item });
```

Reply:
and remove the "this" from the call when the event is emitted.